### PR TITLE
misc changes to sh411

### PR DESCRIPTION
**Pull request by Icenowy**
The author of this commit finished her surgery at 411 on Mar 22 and left
hospital now. This commit contains misc changes to this page according
to the experience of the author.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>

------------------------

P.S. Some QR codes (standard QR codes, but relies WeChat to work) are now posted in the rooms at 411, for necessary knowledge. Should I post them on mtf.wiki?

### CODE REVIEW

File: content/zh-cn/docs/srs/china/sh411/index.md [hunk 1]
```diff
@@ -21,12 +21,12 @@ keywords:
...
 所在医院：[海军军医大学第一附属医院（上海长海医院）虹口院区](https://amap.com/place/B0FFKP410J)[^2]
 
-赵烨德医生是国内相对有经验的 SRS 医生，使用阴囊皮瓣作为阴道，阴茎皮瓣成形阴唇。411 近两年得到改进的手术主要优点在于价格相对较低，对通模具要求较少，敏感度保留好，且能实现各个结构的外观成形；缺点主要在于外观尚欠自然，深度较浅。同时，411 的手术体验相对差，需要禁食一周、卧床两周。
-
-实行 0 深度的外阴成型手术可以提供鉴定更改身份证
+赵烨德医生是国内相对有经验的 SRS 医生，使用阴囊皮瓣作为阴道，阴茎皮瓣成形阴唇。411 近两年得到改进的手术主要优点在于价格相对较低，对通模具要求较少，敏感度保留好，且能实现各个结构的外观成形；缺点主要在于仅使用皮瓣作为阴道材料，深度较浅，且当阴茎+阴囊皮瓣不足时将需要腹股沟/大腿皮瓣作为补充，此时将增大手术痛苦并延长术后恢复时间。同时，411 的手术体验相对差，需要禁食一周、卧床两周。
 
 手术需要停止 HRT 一个月
 
+当前由于国际形势，赵博处 MtF SRS 排队队列较长（即使是理想状况下也需排队两三个月），且受上海疫情影响当前已排队的患者结果无法预期。
```
Comment (septs) [inline content/zh-cn/docs/srs/china/sh411/index.md:28]: 需要添加时效性声明

**Comment by Icenowy** [comment]
图片
The QR codes I mentioned

**Comment by saeziae** [comment]
> \[图片\]\
>The QR codes I mentioned

Is it possible to convert to text(markdown) files?

**Comment by Icenowy** [comment]
> > [图片]
> > The QR codes I mentioned
> 
> Is it possible to convert to text(markdown) files?

It's possible, but the problem is the dependency to WeChat.

**Comment by saeziae** [comment]
> > > [图片]
> > > The QR codes I mentioned
> > 
> > 
> > Is it possible to convert to text(markdown) files?
> 
> It's possible, but the problem is the dependency to WeChat.

wtf how can such kind of content depend on wechat haaiyaa

**Comment by Icenowy** [comment]
> > > > [图片]
> > > > The QR codes I mentioned
> > > 
> > > 
> > > Is it possible to convert to text(markdown) files?
> > 
> > 
> > It's possible, but the problem is the dependency to WeChat.
> 
> wtf how can such kind of content depend on wechat haaiyaa

oh there's a fallback viewer in HTML.

Then I will try to convert this to md, and add mtf related links to wiki

**Comment by Icenowy** [comment]
links added, too lazy to consider ftm things.

**Comment by Icenowy** [comment]
ping?

**Comment by saeziae** [comment]
Pinging mtf.wiki [172.67.144.156] with 32 bytes of data:
Reply from 172.67.144.156: bytes=32 time=199ms TTL=52
Reply from 172.67.144.156: bytes=32 time=200ms TTL=52
Reply from 172.67.144.156: bytes=32 time=216ms TTL=52
Reply from 172.67.144.156: bytes=32 time=200ms TTL=52

Ping statistics for 172.67.144.156:
    Packets: Sent = 4, Received = 4, Lost = 0 (0% loss),
Approximate round trip times in milli-seconds:
    Minimum = 199ms, Maximum = 216ms, Average = 203ms

File: content/zh-cn/docs/srs/china/sh411/index.md [hunk 2]
```diff
@@ -21,12 +21,12 @@ keywords:
...
 所在医院：[海军军医大学第一附属医院（上海长海医院）虹口院区](https://amap.com/place/B0FFKP410J)[^2]
 
-赵烨德医生是国内相对有经验的 SRS 医生，使用阴囊皮瓣作为阴道，阴茎皮瓣成形阴唇。411 近两年得到改进的手术主要优点在于价格相对较低，对通模具要求较少，敏感度保留好，且能实现各个结构的外观成形；缺点主要在于外观尚欠自然，深度较浅。同时，411 的手术体验相对差，需要禁食一周、卧床两周。
-
-实行 0 深度的外阴成型手术可以提供鉴定更改身份证
+赵烨德医生是国内相对有经验的 SRS 医生，使用阴囊皮瓣作为阴道，阴茎皮瓣成形阴唇。411 近两年得到改进的手术主要优点在于价格相对较低，对通模具要求较少，敏感度保留好，且能实现各个结构的外观成形；缺点主要在于仅使用皮瓣作为阴道材料，深度较浅，且当阴茎+阴囊皮瓣不足时将需要腹股沟/大腿皮瓣作为补充，此时将增大手术痛苦并延长术后恢复时间。同时，411 的手术体验相对差，需要禁食一周、卧床两周。
 
 手术需要停止 HRT 一个月
 
+当前（2022 年 4 月）由于国际形势，赵博处 MtF SRS 排队队列较长（即使是理想状况下也需排队两三个月），且受上海疫情影响当前已排队的患者结果无法预期。
```
Comment (saeziae) [inline content/zh-cn/docs/srs/china/sh411/index.md:28]: > 国际形势

?

File: content/zh-cn/docs/srs/china/sh411/index.md [hunk 3]
```diff
@@ -161,9 +163,17 @@ keywords:
...
+- [术后用药](http://qr61.cn/oWtfl1/qXLtGrb)
+- [出院流程](http://qr61.cn/oWtfl1/qGkmRMx)
+
 [^1]: 医生叫赵烨德，不叫赵博。
 [^2]: 原中国人民解放军第 411 医院。
 [^3]: 可以直接搜索进入，亦可进入微信公众号「上海长海医院」后选择「医疗服务」-「线下医疗」进入。
 [^4]: 术前针注射时间取决于手术时间。如果手术安排在下午，术前针也会推迟注射。
 [^5]: 即产褥垫。
 [^6]: 仅可于住院期间查询，在医院为其办理出院后（在出院结清费用之前，不需要其本人参与）将无法查询。
+[^7]: 疑似用途见赵烨德作为通讯作者的论文 《应用尿道海绵体黏膜组织瓣构建会阴前庭的临床研究》。
```
Comment (saeziae) [inline content/zh-cn/docs/srs/china/sh411/index.md:179]: 有无原文链接或原文相关摘录？

File: content/zh-cn/docs/srs/china/sh411/index.md [hunk 4]
```diff
@@ -161,9 +163,17 @@ keywords:
 
 - 在添加就诊人时，如果没有就诊卡，则此就诊人仅能预约挂号。医院会在初次就诊时提供就诊卡。获得此卡后可在小程序内绑定就诊卡以查询住院费用[^6]、预约病案等。
 
+## 官方宣教链接
+
+以下链接指向科室官方发布的宣教信息（以二维码形式发布在病房内）：
+- [MTF 手术](http://qr61.cn/oWtfl1/qnGi3dj)
```
Comment (septs) [inline content/zh-cn/docs/srs/china/sh411/index.md:169]:
```suggestion
- [MtF 手术](http://qr61.cn/oWtfl1/qnGi3dj)
```

File: content/zh-cn/docs/srs/china/sh411/index.md (hunk 3 above)
Comment (Icenowy) [reply content/zh-cn/docs/srs/china/sh411/index.md:179]: 我可以给出 doi （

**Pull request merged**